cpp/open3d/geometry/TriangleMesh.cpp

```diff
@@ -293,6 +293,16 @@ std::shared_ptr<TriangleMesh> TriangleMesh::FilterSmoothSimple(
             std::swap(mesh->vertex_colors_, prev_vertex_colors);
         }
     }
+    if (filter_vertex) {
```

Comment:
Is this a bugfix? Can you move it to a separate PR, since its unrelated to the filament upgrade?

Reply:
yes, i have this error after upgrade to ubuntu 22.04 or use armv8